source/core/Date.cpp

```diff
@@ -301,7 +301,7 @@ namespace Vireo {
             gmtime_r(&time, &tm);
         else
             localtime_r(&time, &tm);
-        _timeZoneOffset = int(isUTC ? 0 : tm.tm_gmtoff);
+        _timeZoneOffset = static_cast<int>(isUTC ? 0 : tm.tm_gmtoff);
```

Comment:
Nothing wrong with a constructor cast here.

Reply:
Yes, but let's use the static_cast to keep the code using only one kind of casting.